### PR TITLE
🌐 Add Chinese translation for Tutorial - Extra Models

### PR DESCRIPTION
**Pull request by waynerv**
@maoyibo @Dustyposa @RunningIkkyu Please take a look.

### CODE REVIEW

File: docs/zh/docs/tutorial/extra-models.md [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+user_in = UserIn(username="john", password="secret", email="john.doe@example.com")
+```
+
+然后我们调用：
+
+```Python
+user_dict = user_in.dict()
+```
+
+现在我们有了一个数据位于变量 `user_dict` 中的 `dict`（它是一个 `dict` 而不是 Pydantic 模型对象）。
```
Comment (maoyibo) [inline docs/zh/docs/tutorial/extra-models.md:44]: 现在我们有了一个包含 `user_dict` 变量中数据的 `dict`

**Review by maoyibo — COMMENTED** [review]
It looks pretty good.

**Review by maoyibo — COMMENTED** [review]
It looks good.

**Review by Laineyzhang55 — APPROVED** [review]

**Review by yanever — APPROVED** [review]

**Review by maoyibo — APPROVED** [review]

**Comment by tiangolo** [comment]
Awesome! Thank you @waynerv :bow: :rocket: 

And thanks for the reviews @maoyibo , @yanever , @Laineyzhang55 :cake: :coffee:

**Pull request merged**